src/docfx/restore/lock/DependencyLock.cs

```diff
@@ -51,12 +51,7 @@ public static DependencyLockModel Load(string docset, SourceInfo<string> depende
                 }
             }
 
-            var (_, content, _) = RestoreMap.GetRestoredFileContent(docset, dependencyLockPath);
-
-            if (string.IsNullOrEmpty(content))
```

Comment:
this shouldn't use `string.IsNullOrEmpty`

Reply:
so to fix this line, changed so many files..? can we split the other changes to other PRs?

Reply:
sure